samples/sample_common/include/sample_utils.h

```diff
@@ -346,10 +372,12 @@ class ExtBufHolder : public T
         //remove all existing extension buffers
         ClearBuffers();
 
+        const auto ref_ = ExtParamAccessor<T>(ref);
```

Comment:
why it is better than previous code?

Reply:
Which previous? With two version of ExtBufHolder& operator=() ?It's better because now we have one version.